Resources/Prototypes/_Impstation/Entities/Clothing/Neck/cloaks.yml

```diff
@@ -80,3 +80,14 @@
     sprite: _Impstation/Clothing/Neck/Cloaks/hosgreatcoat.rsi
   - type: StealTarget
     stealGroup: HeadCloak
+
+- type: entity
+  parent: [ClothingNeckBase, BaseCommandContraband]
```

Comment:
if you parent this to just ClothingCloakCmo (so, `parent: ClothingCloakCmo` with no need for square brackets) then you don't need to copy components like the StealTarget and it'll automatically get adjusted if the base item ever gets anything added to it